Header header
uint8 accelerometerCalib
uint8 gyroscopeCalib
vector3D linear
vector3D rotation